main.go

```diff
@@ -30,6 +37,9 @@ func main() {
 	p := plugin.Plugin{
 		Name:    "trustedcoin",
 		Version: "v0.2.5",
+		Options: []plugin.Option{
+			{"trustedcoin-network", "string", nil, "type of the network testnet or bitcoin"},
```

Comment:
That `nil` there is where you should put the default, see https://godoc.org/github.com/fiatjaf/lightningd-gjson-rpc/plugin#Option.